smnmna99/Connect
# Add Docker and Docker Compose for 'connect' project with MySQL and phpMyAdmin

## Description
This pull request introduces Docker and Docker Compose support for the 'connect' project, streamlining the development environment setup. The Docker Compose configuration includes containers for the main application, MySQL, and phpMyAdmin.

## Changes Made
- Added Dockerfile for the main application.
- Created Docker Compose file (`docker-compose.yml`) to orchestrate containers.
- Configured MySQL container for database support.
- Integrated phpMyAdmin container for database management.
- Update the "application.properties" file to establish a connection between the application and the database.

## How to Test
1. Ensure Docker and Docker Compose are installed on your system.
2. Clone this branch and navigate to the project directory.
3. Run `docker-compose up` to start the containers.
4. Access the application at [http://localhost:8501].
5. Access phpMyAdmin at [http://localhost:8081] .
6. promote connectservice2 to superuser  use this link [https://tableplus.com/blog/2018/10/how-to-create-a-superuser-in-mysql.html].
 
## Additional Notes.
- Feel free to customize the Docker Compose configuration for your specific needs.

This enhancement aims to simplify the setup process and enhance collaboration. Please review and provide feedback.

Thank you!
